engine/schema/resources/META-INF/cloudstack/core/spring-engine-schema-core-daos-context.xml

```diff
@@ -220,6 +220,7 @@
   <bean id="nicSecondaryIpDaoImpl" class="com.cloud.vm.dao.NicSecondaryIpDaoImpl" />
   <bean id="nicIpAliasDaoImpl" class="com.cloud.vm.dao.NicIpAliasDaoImpl" />
   <bean id="objectInDataStoreDaoImpl" class="org.apache.cloudstack.storage.db.ObjectInDataStoreDaoImpl" />
+  <bean id="ontapSvmDaoImpl" class="com.cloud.network.dao.IpReservationDaoImpl" />
```

Comment:
Can we leave the IP Reservation functionality generic?  (Not mention ontap)

Reply:
Done